### DISCUSSION TITLE
Generate shorthands visually

### DISCUSSION THREAD

**Discussion by GrimPixel** [grascii/grascii#83] [Ideas]
I am thinking of a new program that generates Gregg shorthand visually, like in SVG. I don't know if you would decide to have a try.

**Comment by chanicpanic** [comment]
I've played with such a concept in https://github.com/grascii/editor. Although, I'm not actively working on it.

**Comment by chanicpanic** [comment]
You can now generate Gregg Shorthand with the [font](https://github.com/grascii/font).
